app/components/game-card.js

```diff
@@ -12,9 +12,7 @@ export default Ember.Component.extend({
 
   attributeBindings: ['draggable'],
 
-  classNames: ['game-card'],
-
-  classNameBindings: ['isTapped:tapped', 'readOnly::cursor-move'],
+  classNameBindings: ['readOnly::cursor-move'],
```

Comment:
Apparently `position: fixed` children of elements with a `transform` property set on them do not act as expected. I think it's like they're `position: static` or something.